.eslintrc.js

```diff
@@ -17,7 +21,8 @@ module.exports = {
     "plugin:@typescript-eslint/recommended", // Uses the recommended rules from @typescript-eslint/eslint-plugin
     "plugin:react-hooks/recommended",
     "prettier",
-    "plugin:prettier/recommended" // Enables eslint-plugin-prettier and eslint-config-prettier. This will display prettier errors as ESLint errors. Make sure this is always the last configuration in the extends array.
+    "plugin:prettier/recommended", // Enables eslint-plugin-prettier and eslint-config-prettier. This will display prettier errors as ESLint errors. Make sure this is always the last configuration in the extends array.
+    "plugin:storybook/recommended" 
```

Comment:
So message above recommends, that prettier plugin is the last one. Lets do that, not to risk random issues ?

Reply:
Done